tests/synchronization2_tests.cpp

```diff
@@ -28,8 +28,10 @@
 
 #include "extension_layer_tests.h"
 #include "synchronization2_tests.h"
+#include "vk_layer_config.h"
 
 void Sync2Test::SetUp() {
+    SetEnvironment("VK_SYNC2_FORCE_ENABLE", "1");
```

Comment:
Oh yeah. Running the tests in parallel... This code gets run at the start of every tests case. So presumably it could run in multiple threads concurrently.  It seems like a local static mutex would make it do the right thing. Or am I missing something?

Reply:
see new push.